src/jit/lclvars.cpp

```diff
@@ -133,14 +133,13 @@ void                Compiler::lvaInitTypeRef()
     // Change the compRetNativeType if we are returning a struct by value in a register
     if (!hasRetBuffArg && varTypeIsStruct(info.compRetNativeType))
     {
-#ifdef _TARGET_ARM_
-        // TODO-ARM64-NYI: HFA
+#if FEATURE_MULTIREG_RET && defined(FEATURE_HFA)
```

Comment:
Isn't FEATURE_HFA implies FEATURE_MULTIREG_RET ? why do we need two conditions above?

In fact in target.h we should have such a check?